### DISCUSSION TITLE
Problem with verification of email adress

### DISCUSSION THREAD

**Discussion by tabatakenshiro** [RosettaCommons/rosetta#239] [ROSIE] [answered]
Hi. I have a problem with verification ''Your GitHub account does not have verified academic email address!''.
Can you help my with it?
My academic email address: [tabata-k@ncnp.go.jp]

**Comment by lyskov** [comment, marked as answer]
— Thank you for letting us know! This should be fixed now, if you still have issues logging-in please feel free to re-open this issue.
